runelite-client/src/main/java/net/runelite/client/plugins/itemstats/ItemStatOverlay.java

```diff
@@ -151,6 +229,27 @@ else if (group == WidgetInfo.EQUIPMENT_INVENTORY_ITEMS_CONTAINER.getGroupId()
 					tooltipManager.add(new Tooltip(tooltip));
 				}
 			}
+
+			Potion p = Potion.get(itemId);
+
+			if (p != null)
+			{
+				if (p.durationRanges.length == 1)
+				{
+					// Only show "Duration: <time>" if there is one tooltip
+					final String durationToolTip = buildDurationString(p.durationRanges[0].lowestDuration);
+					tooltipManager.add(new Tooltip(durationToolTip));
+				}
+				else
+				{
+					// List the effect names and their duration (ranges)
+					for (PotionDurationRange durationRange : p.durationRanges)
+					{
+						final String durationToolTip = buildDurationString(durationRange);
+						tooltipManager.add(new Tooltip(durationToolTip));
```

Comment:
It would probably be better to build one tooltip here with newlines (`<br>`) and then add that.